html/js/display-product.js

```diff
@@ -35,22 +35,22 @@ class RobotoffAsker extends HTMLElement {
         .hidden { display: none; }
       </style>
       <div class="row hidden zoomrow" id="zoomrow">
-        <div class="medium-12 columns">
+        <div class="small-12 columns">
```

Comment:
```suggestion
        <div class="small-10 columns">
```
Re-enable the thumbnail. 10/12 columns seems to be enough for the text.